src/components/core/components/Card/Card.tsx

```diff
@@ -50,7 +50,7 @@ export default function Card(props: Props) {
   }
 
   return (
-    <StyledCardMaterial onClick={handleClick} clickable={!!onSelect} disabled={disabled}>
+    <StyledCardMaterial onClick={handleClick} $clickable={!!onSelect} disabled={disabled}>
```

Comment:
$clickable property does not exists

Reply:
the $ makes it so that the react dom does not yell at us about it

Reply:
we can fix it directly in the styled component

Reply:
pushing these changes now but will do

Reply:
@seeden sorry about all of the eh changes.
this is my first time using react and typescript

Reply:
Just figured out how to fix it the right way